AK/Platform.h

```diff
@@ -47,6 +47,10 @@
 #    define AK_OS_LINUX
 #endif
 
+#if defined(__GLIBC__)
+#   define AK_LIBC_GLIBC
+#endif
```

Comment:
Let's move this after AK_COMPILER_* instead of mixing it with the AK_OS_* macros